modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/test/java/dev/galasa/framework/api/ras/internal/routes/TestResultNamesRoute.java

```diff
@@ -83,6 +83,8 @@ private String generateExpectedJSON (List<IRunResult> mockInputRunResults, boole
 		resultNames.add("EnvFail");
 		resultNames.add("Ignored");
 		resultNames.add("Cancelled");
+		resultNames.add("Requeued");
```

Comment:
Wondering if the same data is in the CLI or whether it doesn't restrict inputs from users in terms of state queries.